packages/editor/src/components/provider/index.native.js

```diff
@@ -124,6 +125,12 @@ class NativeEditorProvider extends Component {
 				this.props.updateSettings( theme );
 			}
 		);
+
+		this.subscriptionParentUpdateCapabilities = subscribeUpdateCapabilities(
+			( payload ) => {
+				this.updateCapabilitiesAction( payload );
+			}
+		);
```

Comment:
It would be good to remove the subscription on `componentWillUnmount` and clean up the logs.